client/src/components/ImageCarousel.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ImageStrip from './ImageStrip';
+import { ImageStripWrapper, MainImageWrapper, MainImage } from './styles';
+
+const ImageCarousel = ({ images, mainImage, onImageClick }) => {
```

Comment:
Love how simple this component is, super easy to read/understand and well coded. You could actually simplify it even further by replacing:
=> {
  return (
    <html>
  );
}

with just:
=> (
    <html>
);

Also, you might want to add an image description to your alt property on the image element as opposed to leaving it blank (""). It might be difficult since that main image is different depending on the game but maybe something like "main game image"? I know AirBnB requires an alt but I also think I read that it's good practice to use an alt for an image description for one day when your code is deployed to the web. I think it helps with search or some aspect of the webpage to have that alt property.

Overall, super solid component and you made it surprisingly simple for how much it actually does with all the photos

Reply:
oh cool, i didn't know that AirBnB uses the alt attribute like that, that's good to know!